Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs

```diff
@@ -216,8 +242,42 @@ private void AddConnectedPersonInformationSection(ColumnDescriptor col, List<Con
         col.Item().PaddingBottom(10);
     }
 
+    private void AddAdditionalIdentifiersSection(ColumnDescriptor col, IEnumerable<Model.Identifier> additionalIdentifiers)
+    {
+        col.Item().Text("Additional Identifiers").Bold().FontSize(12);
+        col.Item().PaddingBottom(10);
+
+        if (additionalIdentifiers.Any())
+        {
+            foreach (var identifier in additionalIdentifiers.Where(i => i.Scheme != "VAT" && i.Uri != null))
+            {
+                col.Item().LineHorizontal(1);
+
+                if (!string.IsNullOrEmpty(identifier.Scheme))
+                    col.Item().Element(container => AddTwoColumnRow(container, "Scheme", identifier.Scheme));
+
+                if (!string.IsNullOrEmpty(identifier.Scheme))
+                    col.Item().Element(container => AddTwoColumnRow(container, "Legal name", identifier.LegalName));
+
+                if (!string.IsNullOrEmpty(identifier.Scheme))
+                    col.Item().Element(container => AddTwoColumnRow(container, "Uri", identifier.Uri));
+            }
```

Comment:
@JBaigGoaco just one question, don't we need to show Primary Identifier?

Reply:
I think the Primary identifier is the VAT number displayed at the top of the PDF. or maybe i am wrong

Reply:
@JBaigGoaco VAT is never the primary identifier.

Reply:
![image](https://github.com/user-attachments/assets/2eacc9e5-a8d1-4dee-81a4-72a293a769d9)

Yes we are displaying the Primary Identifier.